bin/tessel-2.js

```diff
@@ -183,6 +185,10 @@ makeCommand('wifi')
     metavar: 'PASSWORD',
     help: 'Set the password of the network to connect to'
   })
+  .option('off', {
+    flag: true,
+    help: 'Disable the wireless network'
```

Comment:
That's a good question. I actually haven't tried this with an access point enabled. It may cancel both station and access point connections!